### PR TITLE
Create 2024-03-22-Civic-Tech-Career-Resources.md

### PR DESCRIPTION
**Pull request by laneymangan**
Prepping our Civic Tech Career Resources blog post to be posted on Monday

### CODE REVIEW

**Comment by AnthonyRollins** [comment]
@laneymangan FYI - the excerpt section appears to have the language from creating blog posts - see screenshot

Screenshot 2024-03-22 at 4 46 35 PM

**Comment by laneymangan** [comment]
@AnthonyRollins thanks! I haven't filled in the front matter yet so when I do I'll be sure to update that

File: _posts/2024-03-22-Civic-Tech-Career-Resources.md
Comment (vykster) [reply _posts/2024-03-22-Civic-Tech-Career-Resources.md:62]: hmm, it looks like they've moved. @mkonda are you in Texas now?!

**Comment by vykster** [comment]
@laneymangan apologies for the late request, but can we removed the justice tech newsletter? it looks like it hasn't been active in almost a year.

**Comment by laneymangan** [comment]
@compilerla/engineering this is ready for review/approval!

File: _posts/2024-03-22-Civic-Tech-Career-Resources.md
Comment (thekaveman) [inline _posts/2024-03-22-Civic-Tech-Career-Resources.md:187]: Few things here:

* This shouldn't be a heading (the `##` text makes it a heading 2)
* This is the link to the old Mailchimp list, should be pointing to the new Hubspot list: https://share.hsforms.com/1yPW2aSteQS6VTeGDEyFq9gqq0e7
* I feel like the text `last but not least` is a little unnecessary (IMHO)

**Review by thekaveman — CHANGES_REQUESTED** [review]

File: _posts/2024-03-22-Civic-Tech-Career-Resources.md
Comment (laneymangan) [reply _posts/2024-03-22-Civic-Tech-Career-Resources.md]: @thekaveman thanks, made the edits.

**Review by thekaveman — APPROVED** [review]
Thanks for those fixed @laneymangan!

This looks good to me, but I think @AnthonyRollins and/or @vykster should approve too? 

Also, totally unrelated to this post, but it seems like we could tighten up the styles for blog posts, there's a lot of empty space i.e. between headings and sections. Will file an issue for that.

**Comment by laneymangan** [comment]
> 
> This looks good to me, but I think @AnthonyRollins and/or @vykster should approve too?

FYI I'm about to do another run-through to double-check that all the links are accurate so no need for others to spend time on that but other edits/approvals are welcome!
> 
> Also, totally unrelated to this post, but it seems like we could tighten up the styles for blog posts, there's a lot of empty space i.e. between headings and sections. Will file an issue for that.

Thanks! That would be great

**Comment by thekaveman** [comment]
@laneymangan some general practice guidance for PRs on GitHub: when a PR is opened for review, typically the PR should be "ready" from the author's perspective. The only changes to the PR after review should be to address review feedback. This is because every change dismisses previous reviews/approvals, and the whole cycle starts over again.

There is an option to open a PR in "Draft" mode, which is a great way to share an early version while you are still working on it. And then once you've done all your checks and edits and etc., opening for review signals to everyone that the PR is assumed to be in a final state and could be merged as-is if approved. 

When you go to open the PR, the green button has a drop-down where you can select "Create a draft pull request":

image

Then once you're ready for review, you can click that button in the bottom of the PR:

image

This notifies all reviewers and again signals that the author thinks this PR is ready to be merged.

**Comment by laneymangan** [comment]
> @laneymangan some general practice guidance for PRs on GitHub: when a PR is opened for review, typically the PR should be "ready" from the author's perspective. The only changes to the PR after review should be to address review feedback. This is because every change dismisses previous reviews/approvals, and the whole cycle starts over again.
> 
> There is an option to open a PR in "Draft" mode, which is a great way to share an early version while you are still working on it. And then once you've done all your checks and edits and etc., opening for review signals to everyone that the PR is assumed to be in a final state and could be merged as-is if approved.
> 
@thekaveman Ahhh yes this is exactly what I need! As I'm still getting used to this process I've found it helpful to start the Github process, get input, make changes, etc until it's where we want it to be, so the draft option seems perfect for that. Ideally, the workflow will be that we will do all editing in google docs before starting the PR, but for now, it's good to know I can start a draft when needed. Thanks!!

**Comment by thekaveman** [comment]
@laneymangan I should have noted that you can also put an existing PR into draft state using this somewhat hidden link up near the reviewers pane on the right:

image

You can always go back and forth between draft/not draft for any PR.

**Comment by laneymangan** [comment]
@thekaveman any way we can merge this without Vyki's approval? (at Vyki's request) CC @AnthonyRollins

**Review by thekaveman — APPROVED** [review]

**Pull request merged**
